mm2src/mm2_main/src/lp_swap.rs

```diff
@@ -218,6 +218,24 @@ pub fn broadcast_swap_message_every<T: 'static + Serialize + Clone + Send>(
     spawn_abortable(fut)
 }
 
+/// Spawns the loop that broadcasts message every `interval` seconds returning the AbortOnDropHandle
+/// to stop it
+pub fn broadcast_swap_message_every_with_initial_delay<T: 'static + Serialize + Clone + Send>(
+    ctx: MmArc,
+    topic: String,
+    msg: T,
+    interval: f64,
+    p2p_privkey: Option<KeyPair>,
+) -> AbortOnDropHandle {
+    let fut = async move {
+        loop {
+            Timer::sleep(interval).await;
+            broadcast_swap_message(&ctx, topic.clone(), msg.clone(), &p2p_privkey);
```

Comment:
perhaps it's better to broadcast first, then wait

Reply:
There is already the `broadcast_swap_message_every` function for this which was normally used. For the watcher message I needed a special case because it is already broadcasted once right after the taker payment is sent [here](https://github.com/KomodoPlatform/atomicDEX-API/blob/3a3aef4a91e930ed87fcc2b833f49c480d664c08/mm2src/mm2_main/src/lp_swap/taker_swap.rs#L1590) to make sure that at least one watcher message is sent if the node goes offline at this point. Afterwards, it should be broadcasted periodically again, but after waiting for `interval` seconds first to prevent broadcasting two watcher messages consecutively.